bears/r/RLintBear.py

```diff
@@ -23,7 +23,8 @@ class RLintBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     REQUIREMENTS = {RscriptRequirement(package='lintr', flag='-e',
-                                       repo='http://cran.rstudio.com')}
+                                       repo='http://cran.rstudio.com',
+                                       version='>=3.1.1')}
```

Comment:
@damngamerz These tests always get skipped :smile: Moreover there is no need for tests here.